spec/mailers/user_mailer_spec.rb

```diff
@@ -300,7 +300,6 @@ def expect_email_body_to_have_help_and_contact_links
     end
 
     it 'renders the body' do
-      reset_text = t('user_mailer.account_reset_granted.cancel_link_text')
```

Comment:
I wonder if we ought to adapt this into an assertion?

```rb
expect(mail.html_part.body).to have_content(reset_text)
```

Reply:
turns out that text is no longer in the template, so going to leave it removed